frontend/components/dashboard/DashboardValidatorOverview.vue

```diff
@@ -0,0 +1,176 @@
+  const elValue = formatFunction(value.el)
+  return {
+    label: $t(`statistics.${key}`),
+    value: `${clValue} (CL) ${elValue} (EL)`,
+  }
+}
+
+const rewardFull = computed(() => converter.value.weiToValue(
+  totalElCl(overview.value?.rewards.last_30d ?? {
+    cl: '0',
+    el: '0',
+  }), { addPlus: true }).fullLabel)
+const reward = computed(() => converter.value.weiToValue(
+  totalElCl(overview.value?.rewards.last_30d ?? {
+    cl: '0',
+    el: '0',
+  }), { addPlus: true }).label)
+
+const validatorsOffline = computed(() => overview.value?.validators.offline ?? 0)
+const validatorsOnline = computed(() => overview.value?.validators.online ?? 0)
+const validatorsInfos = computed(() =>
+  [
+    {
+      label: $t('dashboard.validator.overview.validators_balance.balance_staked'),
+      value: formatValueWei(overview.value?.balances.staked_eth ?? 0),
+    },
+    {
+      label: $t('dashboard.validator.overview.validators_balance.balance_effective'),
+      value: formatValueWei(overview.value?.balances.effective ?? 0),
+    },
+  ],
+)
+
+const dialog = useDialog()
+const { dashboardKey } = useDashboardKey()
+const { getDashboardLabel } = useUserDashboardStore()
+const openValidatorModal = () => {
+  dialog.open(DashboardValidatorSubsetModal, {
+    data: {
+      context: 'dashboard',
+      dashboardKey: dashboardKey.value,
+      dashboardName: getDashboardLabel(dashboardKey.value, 'validator'),
+      timeFrame: 'last_24h',
+    },
+  })
+}
+
+const efficiencyInfos = computed(() =>
+  TimeFrames.map(k => ({
+    label: $t(`statistics.${k}`),
+    value: formatPercent(overview.value?.efficiency[k] ?? 0),
+  })),
+)
+
+const rewardsInfos = TimeFrames.map(k =>
+  createInfo(k, overview.value?.rewards[k] ?? {
+    cl: '0',
+    el: '0',
+  }, formatValueWei),
+)
+
+const apr = computed(() => formatPercent(totalElClNumbers(overview.value?.apr.last_30d ?? {
+  cl: 0,
+  el: 0,
+})),
+)
+
+const aprInfos = TimeFrames.map(k =>
+  createInfo(k, overview.value?.apr[k] ?? {
+    cl: 0,
+    el: 0,
+  }, formatToPercent),
+)
+</script>
+
+<template>
+  <div class="container">
+    <DashboardValidatorOverviewItem
+      :infos="validatorsInfos"
+      :title="$t('dashboard.validator.overview.online_validators')"
+    >
+      <span :class="{ positive: validatorsOnline }">
+        {{ validatorsOnline }}
+      </span> |
+      <span :class="{ negative: validatorsOffline }">
+        {{ validatorsOffline }}
+      </span>
+      <BcButtonIcon
+        :sr-text="$t('dashboard.validator.overview.open_validator_overview_modal')"
+        @click="openValidatorModal"
+      >
+        <FontAwesomeIcon
+          class="link optical-correction"
+          :icon="faArrowUpRightFromSquare"
+        />
+      </BcButtonIcon>
+      <template #additionalInfo>
+        {{ $t('dashboard.validator.overview.validators_balance.balance_total') }}
+        <span class="bold">
+          {{ formatValueWei(overview?.balances.total ?? 0) }}
```

Comment:
Nitpick: even dough the formatValueWei can handle `number` and `string` the total is of type string so I would default to `'0'`

Reply:
In general I would second that, but in template strings in between `{{ }}` mustache syntax, there is no difference